### PR TITLE
Add the plugin Skript

### PR DESCRIPTION
**Pull request by BaeFell**
- Adds Skript with a config

:+1:

### CODE REVIEW

**Comment by Chnkr** [comment]
:-1: No.

**Comment by ducc** [comment]
:-1: :-1: :-1: :-1: :-1: :-1: :-1: :-1: :-1: :-1: :-1: :-1: :-1: :-1: :-1: :-1: :-1: :-1: :-1: :-1: :-1: :-1: :-1: :-1: :-1: :-1: :-1: :-1: :-1: :-1: :-1: :-1: :-1: :-1: :-1: :-1: :-1: :-1: :-1: :-1: :-1: :-1: :-1: :-1:

**Comment by Nebhay** [comment]
:-1:

**Comment by Yeshael** [comment]
:+1:

Absolutely!!!  I wouldn't be interested in a server without it. Too valuable.

**Comment by Serializator** [comment]
:-1::-1::-1::-1::-1::-1::-1::-1::-1::-1::-1::-1::-1::-1::-1::-1::-1::-1::-1::-1::-1::-1::-1::-1::-1::-1::-1::-1::-1::-1:

**Comment by JamieSinn** [comment]
No.
:-1:

**Comment by BaeFell** [comment]
You're all negative nellies!

**Comment by alaaeelsayed** [comment]
:-1::-1::-1::-1::-1::-1::-1::-1::-1::-1::-1::-1::-1::-1::-1::-1::-1::-1::-1::-1::-1::-1::-1::-1::-1::-1::-1::-1::-1::-1:
Bad idea.. ruins the idea of Spigot

**Comment by connorlurring** [comment]
:-1: no

**Comment by BaeFell** [comment]
@alaaemad15 Says the one with the Skript addon. Oh and this is Spigot.

**Comment by alaaeelsayed** [comment]
@nfell2009 And? why would the creators of spigot use something to help you use the spigot api?

**Comment by BaeFell** [comment]
@alaaemad15 Your messages are making no sense. You're saying Skript ruins the idea of Spigot? Yet you make an addon for Skript. You might wanna reread your posts and make sure they make sense.

**Comment by alaaeelsayed** [comment]
@nfell2009 I meant it ruins the idea of the suggestion of plugins to be added to the server. because skript would be useless.
Like exactly what would they do with Skript? Please tell me.

**Comment by wrong7** [comment]
:+1:

**Comment by ghost** [comment]
@nfell2009 Good job, you actually made me feel mildly annoyed for a minute or two. I must admit however that the 127 people who seem to actually like you seem to be quite odd. The thing is; you must be a troll. There is no way around it, I cannot foresee any way you can be this stupid and still function enough to use a computer. Of course, if you did happen to believe what you're saying then you are such a low-functioning human being that I should just laugh. You're going to live and die a pathetic and ignorant life, surrounding yourself with pathetic ignorant people. It's quite sad, really. Now I understand my "privileges"- being in a decent area with a background of relatively intelligent people I can live not only a much more comforting life but a much more fulfilled one. I will be happy and while you may be happy as well, that will be down to the popular phrase "ignorance is bliss". There is so much more in the world to understand and experience however due to your stupidity you will never realize this and you will be content living a life like a monkey in a cage with low wages and no contact with the interesting parts of the universe. There should be charities for you, you poor human.

tl;dr - :-1: to skript.

**Comment by Ichbinjoe** [comment]
:+1: @Chnkr , :-1: to PR

**Comment by alaaeelsayed** [comment]
@nfell2009 making an addon for Skript doesn't mean that I believe it's better than spigot or it should be used by everyone, if you had you used your brain before committing this you would have noticed that you were submitting Skript which is according to Skript's dev-bukkit is: "a plugin that allows you to customize Minecraft's mechanics with simple scripts written in plain English sentences" And according to spigot's about page Spigot is: "the most widely-used modded Minecraft server software in the world" So... you are asking the creators of the most widely-used modded Minecraft server software in the world to add a plugin that allows you to customize Minecraft's mechanics with simple scripts? to their server? :-1:

**Comment by BaeFell** [comment]
Awh @Chnkr, you do make me laugh. You obivously didn't Google me enough to know me. Gtfo.

@alaaemad15, did you try and be a big boy by posting a big long message because someone else did? At least you posted rather than just editing to make it seem like you didn't make anymore slip ups.

EDIT: Oh and @Chnkr, calling me the low-life yet you just verbally attacked me? Heh.

**Comment by alaaeelsayed** [comment]
@nfell2009 This is completely off topic, you are attempting to start an argument with me although I was proving my point of why Skript shouldn't be added.

**Comment by BaeFell** [comment]
Looks like quite a few people has piss in their cheerios this morning. Ta ta bitchz.

**Comment by alaaeelsayed** [comment]
Ahh, that's why it tasted good.

**Comment by BaeFell** [comment]
@alaaemad15, okay... I'll give you that one lol

**Comment by Yeshael** [comment]
@Chnkr 

Ok, first... do you even OWN a mirror? I mean.... damn, kid! You should seriously consider suing your parents for genetic neglect. 

I take it you were bullied heavily in school, and you feel this is your chance to act all brave since you believe yourself immune online. But ranting and raving about the intellect or abilities of others, so that you can feel better about yourself is truly pathetic, and will not ever make up for being a loser. Now, I'm sure you must see yourself as brilliant. That's excellent. What's your MENSA ID name? I 'd love to look you up and we can compare notes and see how your intellect holds up against others that truly are.

Ok... on a serious note.

How do you like it, little child? Not fun, right?

It's one thing to dislike the proposal and vote against it. I can respect that. Can you not make an "intelligent" rational mature statement without attacking this person's character? You're making a tremendous amount of claims with nothing to base it on. You DO realize how foolish that makes YOU look, right? Making such wild raving claims about someone you don't even know, let alone "all" the people that know him is really lame and childish.

Ok, so now I fully expect your playground-style "Oh yeah!? Well...." response... so go ahead. Although I have better things to do than read it, so it will be wasted effort. But go ahead if if will make you feel better.

**Comment by ghost** [comment]
@Yeshael If you didn't notice yet, it was a silly copypasta from the internet. :+1:

**Comment by Sataniel98** [comment]
:-1:

**Comment by BaeFell** [comment]
@Feuerstern THIS IS CLOSED. GO AWAY.

**Comment by Sataniel98** [comment]
Are you a bull? Am I wearing anything red? No reason to get upset ;)

**Comment by BaeFell** [comment]
@Feuerstern, Sorry hun. I meant to say; unfortunately this is closed <3

**Comment by Cldfire** [comment]
> Saw post on forums that nfell had started a script PR
> Clicked link
> Mind blown

Well, now we know how to get people interested in the server :D

**Comment by BaeFell** [comment]
@Cldfire, I don't do simple. I do funny.
